### PR TITLE
DAT-4566 | Make icons colorable

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/DAT-4566
https://github.com/Wikia/design-system-reference-page/pull/34

- manually removed obsolete groups from SVGs that were interfering with styling
- modified gulp task to:
  - split icons and logos into separate files:
    - `dist/icons.svg`
    - `dist/company.svg`
  - insert `<style>.fill {fill: currentColor;}</style>` into combined icons SVG
  - replace all `fill` attributes (e.g. `fill="#999"`) with `class="fill"`

It allows us to color icons with simple:
```
.icon {
  color: #999;

  &:hover {
    color: #444;
  }
}
```

@Wikia/x-wing

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+save-exact=true
```
Comment (hakubo) [inline .npmrc]: ideally this would be another PR :)

Comment (rogatty) [inline .npmrc]: It's really minor and I don't feel confident with squashing / rebasing / whatever git magic it would require.

**Comment by hakubo** [comment]
lgtk
